src/main/scala/io/github/evis/scalafix/maven/plugin/params/ProjectDependenciesParam.scala

```diff
@@ -1,18 +1,23 @@
 package io.github.evis.scalafix.maven.plugin.params
 
-import org.apache.maven.artifact.Artifact
+import java.util.jar.JarFile
+
+import scala.util.Try
 
-import scala.collection.JavaConverters._
+import org.apache.maven.artifact.Artifact
 
 object ProjectDependenciesParam {
 
-  def apply(projectDependencies: java.util.Set[Artifact]): MojoParam = {
-    val deps = projectDependencies.asScala.toList
+  def apply(projectDependencies: Iterable[Artifact]): MojoParam = {
```

Comment:
```suggestion
  def apply(projectDependencies: Set[Artifact]): MojoParam = {
```

to show intention clearly, that there's no duplicated artifacts